app/views/facility_reservations/edit_admin.html.haml

```diff
@@ -18,7 +18,7 @@
 = content_for :tabnav do
   = render :partial => 'admin/shared/tabnav_product', :locals => {:secondary_tab => 'reservations'}
 
-%h2= @instrument
+%h2= @reservation.product
```

Comment:
This feels a little strange to me. Let's say the admin reservation was originally created on Instrument #1, and I'm on the schedule page for Instrument #2, and click edit. This is going to show me Instrument #1 and then when I submit it takes me back to #2.